### DISCUSSION TITLE
iPhone build

### DISCUSSION THREAD

**Discussion by baulusdev** [jtothebell/fake-08#94] [Ideas]
Will there be an IPhone build in the future? I just found my old jail broken iPhone SE and wanted to know if I could play pico 8 games on it

**Comment by jtothebell** [comment]
While I would like to make an iOS (and tvOS) build at some point in the future, its pretty far from the top of my list. I don't think it would be super hard to do, except for one major road block: z8lua doesn't play nice with LLVM/clang. I'm hoping to figure that out at some point, as that also stands in the way of an OG Xbox build and an Android build. So in short, I hope so eventually, but its not imminent.
